packages/api/lib/granule-remove-from-cmr.js

```diff
@@ -38,9 +38,13 @@ const _removeGranuleFromCmr = async (granule, collectionId) => {
  * @param {Object} params
  * @param {Knex} params.knex - DB client
  * @param {Object} params.pgGranuleRecord - A Postgres granule record
- * @param {string} params.pgCollection - A Postgres Collection record
- * @param {Object} params.granulePgModel - Instance of granules model for PostgreSQL
- * @param {Object} params.granuleDynamoModel - Instance of granules model for DynamoDB
+ * @param {string} [params.pgCollection] - A Postgres Collection record
```

Comment:
From the code, it looks like this is an object with name and version properties, not a string.

Reply:
3ab40c4345